src/test/ui/codemap_tests/repair_span_std_macros.stderr

```diff
@@ -1,8 +1,10 @@
-error[E0282]: unable to infer enough type information about `T`
+error[E0282]: unable to fully infer type(s)
   --> $DIR/repair_span_std_macros.rs:12:13
    |
 12 |     let x = vec![];
-   |             ^^^^^^ cannot infer type for `T`
+   |         -   ^^^^^^ cannot infer type
+   |         |
+   |         annotating the type for the variable `x` would help
```

Comment:
can you add a similar test, but for the case where the thing being assigned is a more complex pattern?

Something like this:

```rust
let (x,) = (vec![],);
```